.vs/ApplicationInsights.AspNetCore/config/applicationhost.config

```diff
@@ -0,0 +1,990 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Whats the content of .vs folder?

Reply:
oh that's my mistake. the .vs folder was previously in the gitignore, but it doesn't match against the new rule. i'll remove it now. thanks